.jvmopts-nightly-cluster-metrics-sigar

```diff
@@ -0,0 +1,11 @@
+-XX:+UseG1GC 
```

Comment:
all those settings can be in the CI default `.jvmopts-ci` IMHO

Reply:
not sure if we really need G1 GC, will try without.